app/controllers/concerns/shopify_app/scopes_verification.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module ShopifyApp
+  module ScopesVerification
+    extend ActiveSupport::Concern
+    include ScopeUtilities
```

Comment:
I wonder if it's worth isolating this code in its own class, do we expect to use it in other places?